src/js/constants/actions.js

```diff
@@ -1,37 +1,37 @@
 'use strict';
 
 module.exports = {
+  ADD_MARK: 'ADD_MARK',
+  CLEAR_HINTS: 'CLEAR_HINTS',
+  CREATE_PIPELINE: 'CREATE_PIPELINE',
   CREATE_SCENE: 'CREATE_SCENE',
-  HINTS_CLEAR: 'HINTS_CLEAR',
+  DELETE_MARK: 'DELETE_MARK',
+  DISPLAY_HINTS: 'DISPLAY_HINTS',
+  EXPAND_LAYERS: 'EXPAND_LAYERS',
   HINTS_ON: 'HINTS_ON',
-  HINTS_DISPLAY: 'HINTS_DISPLAY',
-  MARK_ADD: 'MARK_ADD',
-  MARK_DELETE: 'MARK_DELETE',
-  MARK_SET_PARENT: 'MARK_SET_PARENT',
-  MARK_UPDATE_PROPERTY: 'MARK_UPDATE_PROPERTY',
-  VEGA_INVALIDATE: 'VEGA_INVALIDATE',
-  VEGA_PARSE: 'VEGA_PARSE',
-  SIGNAL_INIT: 'SIGNAL_INIT',
-  SIGNAL_SET: 'SIGNAL_SET',
-  SIGNAL_SET_STREAMS: 'SIGNAL_SET_STREAMS',
-  SIGNAL_UNSET: 'SIGNAL_UNSET',
+  INIT_SIGNAL: 'INIT_SIGNAL',
+  INVALIDATE_VEGA: 'INVALIDATE_VEGA',
+  PARSE_VEGA: 'PARSE_VEGA',
+  REMOVE_LAYERS: 'REMOVE_LAYERS',
   RULES_ADD_SCALE_TO_GROUP: 'RULES_ADD_SCALE_TO_GROUP',
   RULES_ADD_LEGEND_TO_GROUP: 'RULES_ADD_LEGEND_TO_GROUP',
   RULES_ADD_AXIS_TO_GROUP: 'RULES_ADD_AXIS_TO_GROUP',
   RULES_SET_PROPERTY: 'RULES_SET_PROPERTY',
   RULES_DISABLE_PROPERTY: 'RULES_DISABLE_PROPERTY',
   RULES_RESET_PROPERTY: 'RULES_RESET_PROPERTY',
-  PIPELINE_SELECT: 'PIPELINE_SELECT',
-  CREATE_PIPELINE: 'CREATE_PIPELINE',
-  UPDATE_PIPELINE_DATASET: 'UPDATE_PIPELINE_DATASET',
-  EXPAND_LAYERS: 'EXPAND_LAYERS',
-  REMOVE_LAYERS: 'REMOVE_LAYERS',
   SELECT_MARK: 'SELECT_MARK',
-  TOGGLE_LAYERS: 'TOGGLE_LAYERS',
+  SELECT_PIPELINE: 'SELECT_PIPELINE',
   SET_ACTIVE_WALKTHROUGH: 'SET_ACTIVE_WALKTHROUGH',
   SET_ACTIVE_STEP: 'SET_ACTIVE_STEP',
+  SET_PARENT_MARK: 'SET_PARENT_MARK',
+  SET_SIGNAL: 'SET_SIGNAL',
+  SET_SIGNAL_STREAMS: 'SET_SIGNAL_STREAMS',
   SET_WALKTHROUGH: 'SET_WALKTHROUGH',
-  SET_WALKTHROUGH_ON: 'SET_WALKTHROUGH_ON'
+  SET_WALKTHROUGH_ON: 'SET_WALKTHROUGH_ON',
+  TOGGLE_LAYERS: 'TOGGLE_LAYERS',
+  UNSET_SIGNAL: 'UNSET_SIGNAL',
+  UPDATE_PIPELINE_DATASET: 'UPDATE_PIPELINE_DATASET',
+  UPDATE_PROPERTY_MARK: 'UPDATE_PROPERTY_MARK'
```

Comment:
Might be useful to group actions by noun here rather than alphabetically? I think if you're going to lookup an action in this file, you know the object you want to work on and what you want to do with it but are probably unclear of the specific action name.